src/Okta.Sdk.IntegrationTests/OktaClientShould.cs

```diff
@@ -71,6 +71,7 @@ public async Task ThrowApiExceptionForInvalidToken()
             var client = TestClient.Create(new OktaClientConfiguration
             {
                 Token = "abcd1234",
```

Comment:
In this test, we shouldn't enable the flag. Instead, we have to add a valid oktaDomain.